plugin/builtin/buildbaron/build_baron.go

```diff
@@ -154,14 +154,14 @@ func (bbp *BuildBaronPlugin) buildFailuresSearch(w http.ResponseWriter, r *http.
 		bbp.opts.Password,
 	)
 
-	tickets, err := jiraHandler.JQLSearch(jql, 0, -1)
+	results, err := jiraHandler.JQLSearch(jql, 0, -1)
 	if err != nil {
 		message := fmt.Sprintf("%v: %v, %v", JIRAFailure, err, jql)
 		grip.Error(message)
 		plugin.WriteJSON(w, http.StatusInternalServerError, message)
 		return
 	}
-	plugin.WriteJSON(w, http.StatusOK, tickets)
+	plugin.WriteJSON(w, http.StatusOK, results.Issues)
```

Comment:
My opinion is that this is definitely worth keeping an eye on since some of the Jira queries can be long running (> 1 min), but the frequency of them is fairly rare and most of the queries complete in about a second. Linking to a query would be an extra click + page load and I feel would defeat some of what the build baron plugin was trying to accomplish in the first place